compute_tools/src/compute.rs

```diff
@@ -838,7 +845,7 @@ LIMIT 100",
         info!("Downloading to shared preload libraries: {:?}", &libs_vec);
         let mut download_tasks = Vec::new();
         for library in &libs_vec {
-            download_tasks.push(self.download_extension(library));
+            download_tasks.push(self.download_extension(library, true));
```

Comment:
Would heavily recommend against using `join_all` and instead using `tokio::task::JoinSet` for this.

Reply:
Do I need an `Arc(self)` or something?